services/app/src/utils/useWebSockets.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable no-console */
+/* eslint-disable import/prefer-default-export */
+import { useEffect, useState } from 'react';
+
+export const useWebSockets = (endpoint) => {
```

Comment:
Noretusi sita dalyka iskirti i dvi dalis - viena dalis butu kodas atsakingas uz websocketu connection'o valdyma. Greiciausiai tai net neturetu buti hooksas, nes hooksai labiau skirti perpanaudoti koda reacto komponentuose, o websocketo connectiona atsidarineti mums turbut reikes tik viena karta ir paskui ji perpanaudot. Manau, gal turetu buti kazkokia klase, kuri butu atsakinga uz connectiono atidaryma / uzdaryma / event listeneriu valdyma / respons'u parsinima is stringo i JSON ir requesto parsinima is JSON i string.

Kitas komponentas butu jau hooksas, kuriame butu ne websocketu logika, o musu biznio logika - siuo metu turbut tik useriu list'o gavimas is websocketo.

Reply:
Vietoj klases suggestinciau koki nors `context`.
Hooksas manau gali buti listeneriam uzkabineti.

Reply:
done